prompting/miners/hf_miner.py

```diff
@@ -50,28 +49,42 @@ def __init__(self, config=None):
         super().__init__(config=config)
 
         model_kwargs = None
-        if self.config.neuron.load_quantized:
-            bt.logging.info("Loading quantized model...")
+        if self.config.neuron.load_in_8bit:
+            bt.logging.info("Loading 8 bit quantized model...")
             model_kwargs = dict(
                 torch_dtype=torch.float16,
                 load_in_8bit=True,
             )
 
+        if self.config.neuron.load_in_4bit:
+            bt.logging.info("Loading 4 bit quantized model...")
+            model_kwargs = dict(
+                torch_dtype=torch.float32,
+                load_in_4bit=True,
+            )
+
         if self.config.wandb.on:
-            self.identity_tags = ("zephyr_miner",)
+            self.identity_tags = ("hf_miner",)
```

Comment:
auto tagging is a good idea but can we do this somewhere more general where other classes benefit without repetition?

Reply:
Considering that each model has a unique identity and unique specific tag management, I think this solution is not far from optimal. I think that creating a generic solution will be less intuitive than keeping identity management unique to each miner